### DISCUSSION TITLE
Backups are going to "/app/storage/app/LinkAce" directory

### DISCUSSION THREAD

**Discussion by milindpatel63** [Kovah/LinkAce#380] [Support] [answered]
I am running linkace in docker advanced mode...

i have enabled backups and mapped the /app/storage/app/backups folder to my host...

now whenever the run the command

php artisan backup:run

the backups zip files are saving into the path /app/storage/app/LinkAce inside the container....
how do I specify the path to which backups should go? Any env variable?

Also, in the docker example yml file at https://github.com/Kovah/LinkAce/blob/main/docker-compose.production.yml
Backup path is /app/storage/app/backups

While in the example env file at https://github.com/Kovah/LinkAce/blob/main/.env.docker.production
Backup path written in comment is /storage/app/backups

And mine are going to a whole diff directory 😅

**Comment by milindpatel63** [comment, marked as answer]
Also, I can backup manually by running that command, but automatic backups fail for me with this error in my email..

Exception message: fopen(/app/storage/app/LinkAce/2022-02-01-02-00-05.zip): Failed to open stream: Permission denied

**Reply by Kovah** [reply]
Looks like an issue with the default configuration. I will take a look when I have the time.

**Reply by milindpatel63** [reply]
hey...i thought no one was looking into discussions, so I moved this to an issue 😅 like 2 seconds ago...

no worries...

**Reply by Kovah** [reply]
Was likely solved in #381

**Reply by Kovah** [reply]
@milindpatel63 Did the last version solve your issues?

**Reply by alexschomb** [reply]
@Kovah I had the same issue and was able to fix it thanks to your statement that the env variable should be local_backups. So it seems to be fixed, thanks!

Side note: the docs should be updated as well: https://www.linkace.org/docs/v1/configuration/application-backups/

**Reply by Kovah** [reply]
Thanks for the hint, will change that as soon as I can.

**Reply by Kovah** [reply]
Documentation was updated
